commit edd96f9b05e6e665c5228ba28741568ca9d2ca6c
Author: Max Kellermann <max.kellermann@ionos.com>
Date:   Mon Feb 17 08:30:18 2025 +0100

    include/unordered_{map,set}: remove, use `std` directly instead
    
    This is only an alias for the `std` classes anyway.  This effectively
    reverts commit 8e8672047ccc which added these aliases, probably
    because C++11 was new and not all compilers supported it properly, but
    that was 12 years ago and we're onl C++20 meanwhile.
    
    Signed-off-by: Max Kellermann <max.kellermann@ionos.com>

diff --git a/src/mgr/DaemonServer.h b/src/mgr/DaemonServer.h
index beb396115f3..1da225ffb7e 100644
--- a/src/mgr/DaemonServer.h
+++ b/src/mgr/DaemonServer.h
@@ -19,6 +19,7 @@
 #include <map>
 #include <set>
 #include <string>
+#include <unordered_map>
 
 #include "common/ceph_mutex.h"
 #include "common/LogClient.h"
@@ -149,7 +150,7 @@ protected:
   std::set<ConnectionRef> daemon_connections;
 
   /// connections for osds
-  ceph::unordered_map<int,std::set<ConnectionRef>> osd_cons;
+  std::unordered_map<int, std::set<ConnectionRef>> osd_cons;
 
   ServiceMap pending_service_map;  // uncommitted
 
